indico/modules/events/layout/controllers/images.py

```diff
@@ -56,24 +59,36 @@ def _process(self):
         files = request.files.getlist('image')
         num = 0
         for f in files:
+
             filename = secure_filename(f.filename, 'image')
+            mtype = guess_type(filename)[0]
```

Comment:
This uses the extension and is not very reliable. See e.g. `RHLayoutLogoUpload` on how to do a better check for image types.